django/db/models/constraints.py

```diff
@@ -85,6 +85,10 @@ class Deferrable(Enum):
     DEFERRED = 'deferred'
     IMMEDIATE = 'immediate'
 
+    def __repr__(self):
+        # PY39: Remove when dropping support for PY39.
+        return '%s.%s' % (self.__class__.__qualname__, self._name_)
```

Comment:
Perhaps only override for older Pythons?

```suggestion
    if not PY310:
        def __repr__(self):
            return '%s.%s' % (self.__class__.__qualname__, self._name_)
```

Reply:
Agreed.